src/cow-react/modules/limitOrders/utils/toFraction.ts

```diff
@@ -0,0 +1,16 @@
+import { Fraction } from '@uniswap/sdk-core'
+
+export function toFraction(value: string, isInversed = false) {
+  const [quotient, remainder = ''] = value.split('.')
```

Comment:
this is not locale friendly. Probably is safer for you to use a library of some sort...

Reply:
Ok no problem, I've picked this one https://github.com/infusion/Fraction.js but its basically just to parse the input value and then rest is with Fraction from Uniswap

Reply:
You can just use: `split(/\,|\./)`
I don't know any locales that use something besides , and .